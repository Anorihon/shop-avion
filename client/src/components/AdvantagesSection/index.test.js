import React from 'react'
import { render } from '@testing-library/react'
import Advantages from './index'

test('Advantages component snapshot', () => {
  const { asFragment } = render(<Advantages />)

  expect(asFragment()).toMatchSnapshot()
})
