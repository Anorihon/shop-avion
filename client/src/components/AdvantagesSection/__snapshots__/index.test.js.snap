// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Advantages component snapshot 1`] = `
<DocumentFragment>
  <div
    class="advantages-container"
  >
    <h3
      class="advantages-container-header"
    >
      What makes our brand different
    </h3>
    <div
      class="advantages-block"
    >
      <div
        class="advantages-block__item"
      >
        <img
          alt="advantage"
          src="../img/advantages/Delivery.png"
        />
        <h4
          class="advantages-block__item__title"
        >
          Next day as standard
        </h4>
        <p
          class="advantages-block__item__text"
        >
          Order before 3pm and get your order the next day as standard
        </p>
      </div>
      <div
        class="advantages-block__item"
      >
        <img
          alt="advantage"
          src="../img/advantages/Checkmark--outline.png"
        />
        <h4
          class="advantages-block__item__title"
        >
          Made by true artisans
        </h4>
        <p
          class="advantages-block__item__text"
        >
          Handmade crafted goods made with real passion and craftmanship
        </p>
      </div>
      <div
        class="advantages-block__item"
      >
        <img
          alt="advantage"
          src="../img/advantages/Purchase.png"
        />
        <h4
          class="advantages-block__item__title"
        >
          Unbeatable prices
        </h4>
        <p
          class="advantages-block__item__text"
        >
          For our materials and quality you wonâ€™t find better prices anywhere
        </p>
      </div>
      <div
        class="advantages-block__item"
      >
        <img
          alt="advantage"
          src="../img/advantages/Sprout.png"
        />
        <h4
          class="advantages-block__item__title"
        >
          Recycled packaging
        </h4>
        <p
          class="advantages-block__item__text"
        >
          We use 100% recycled to ensure our footprint is more manageable
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
